tools/mason/MasonSearch.chpl

```diff
@@ -65,7 +65,13 @@ proc masonSearch(origArgs : [] string) {
           }
         }  else {
           const ver = findLatest(searchDir + dir);
-          results.push_back(name + " (" + ver.str() + ")");
+          if ver[1] == false {
+            var warning_str = "skipping package " + dir + " as no valid version was found.";
```

Comment:
Since we already emit a warning per bad filename, this seems a bit verbose. I'd suggest we drop this warning.